docs/examples/contrib/sqlalchemy/sqlalchemy_declarative_models.py

```diff
@@ -34,13 +35,21 @@ class Book(UUIDAuditBase):
 sqlalchemy_config = SQLAlchemyAsyncConfig(
     connection_string="sqlite+aiosqlite:///test.sqlite", session_config=session_config
 )  # Create 'async_session' dependency.
-sqlalchemy_plugin = SQLAlchemyInitPlugin(config=sqlalchemy_config)
+sqlalchemy_plugin = SQLAlchemyPlugin(config=sqlalchemy_config)
 
 
 async def on_startup() -> None:
```

Comment:
This whole function can be removed if we change change the `create_all=True` in the `SQLAlchemyConfig`

Reply:
@cofin I think your intention was to just remove the first two lines (the ones that create the tables) and not the entire function because @JorenSix is adding some data here and removing this hook would prevent that.

Reply:
See below, indeed now I create the metadata via the parameter and the data via the on_startup. Thanks for the suggestion!